central/notifications/datastore/internal/writer/writer_test.go

```diff
@@ -0,0 +1,193 @@
+package writer
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/pkg/errors"
+	storeMocks "github.com/stackrox/rox/central/notifications/datastore/internal/store/mocks"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/protoconv"
+	"github.com/stretchr/testify/suite"
+	"go.uber.org/mock/gomock"
+)
+
+var errFake = errors.New("fake error")
+
+func TestNotificationsWriter(t *testing.T) {
+	t.Parallel()
+	suite.Run(t, new(writerTestSuite))
+}
+
+type writerTestSuite struct {
+	suite.Suite
+	mockCtrl *gomock.Controller
+
+	ctx    context.Context
+	store  *storeMocks.MockStore
+	writer Writer
+}
+
+func (s *writerTestSuite) SetupTest() {
+	s.mockCtrl = gomock.NewController(s.T())
+
+	s.ctx = context.Background()
+	s.store = storeMocks.NewMockStore(s.mockCtrl)
+	s.writer = New(s.store)
+}
+
+func (s *writerTestSuite) TearDownTest() {
+	s.mockCtrl.Finish()
+}
```

Comment:
Same comment as in the previous PR regarding [gomock.Controller.Finish](https://pkg.go.dev/github.com/golang/mock/gomock#Controller.Finish) not being required.